docs/book/expressions.md

```diff
@@ -311,6 +312,40 @@ Otherwise, if you want to deploy multiple skills where most of their
 logic is the same, you can keep them as one Litexa project with different
 `DEPLOY` object configurations, thereby avoiding code duplication.
 
+
+## Customizing the Litexa DB Key
```

Comment:
I don't think this is the right place to put this - this isn't related at all to variable types. Can you move it back?

Reply:
Updating to make it clearer how this belongs next to the description of storage scopes.